acm/odh-edge/apps/tensorflow-housing-app/files/test-data.json

```diff
@@ -0,0 +1 @@
+{"data": [[-120.47, 35.74, 9.0, 4267.0, 785.0, 691.0, 3.7303, 162700.0], [-119.88, 34.43, 16.0, 1734.0, 365.0, 391.0, 4.4777, 282500.0], [-117.8, 33.78, 17.0, 4138.0, 805.0, 780.0, 4.7804, 242000.0], [-117.93, 34.15, 14.0, 9610.0, 2005.0, 1907.0, 4.0393, 156800.0], [-120.43, 34.89, 30.0, 1979.0, 342.0, 320.0, 5.0286, 158000.0], [-118.3, 34.18, 13.0, 7174.0, 1997.0, 1872.0, 3.0973, 251900.0], [-122.47, 37.78, 51.0, 1485.0, 386.0, 385.0, 2.7431, 307100.0], [-118.52, 34.22, 35.0, 1275.0, 222.0, 226.0, 5.0282, 195400.0], [-119.27, 35.5, 21.0, 2171.0, 483.0, 450.0, 1.7105, 52100.0], [-119.79, 34.44, 25.0, 1479.0, 314.0, 309.0, 4.1797, 271800.0], [-122.41, 37.61, 42.0, 1602.0, 262.0, 255.0, 5.7398, 336400.0], [-119.4, 36.25, 25.0, 1696.0, 279.0, 291.0, 2.3, 132800.0], [-122.3, 37.9, 35.0, 1102.0, 308.0, 303.0, 2.3946, 141700.0], [-118.27, 33.99, 41.0, 656.0, 162.0, 170.0, 1.8047, 101800.0], [-118.19, 33.98, 34.0, 1022.0, 286.0, 275.0, 2.6042, 156700.0], [-121.75, 36.76, 32.0, 1740.0, 399.0, 389.0, 2.7694, 132400.0], [-120.91, 38.11, 9.0, 3585.0, 680.0, 598.0, 3.636, 133100.0], [-118.24, 34.22, 34.0, 1722.0, 406.0, 371.0, 4.1523, 252000.0], [-116.85, 34.25, 5.0, 5806.0, 1030.0, 219.0, 4.0132, 163100.0], [-121.04, 37.68, 28.0, 1909.0, 398.0, 380.0, 2.3783, 81400.0]]}
```

Comment:
How does this test data get created? Is it generated from the original data set?

Reply:
This was generated from the dataset included in the PR with a python script that I have not included, would you want me to add that aswell? I mostly the same as ahmeds in his PR

Reply:
I would suggest including the script :+1:  Similar to how Ahmed included his one here: https://github.com/opendatahub-io/ai-edge/blob/main/pipelines/models/bike-rentals-auto-ml/test_data_generator.py